src/Runtime/OMTensor.inc

```diff
@@ -344,6 +346,76 @@ void *omTensorGetAllocatedPtr(const OMTensor *tensor) {
   return tensor->_allocatedPtr;
 }
 
+void omTensorPrint(const OMTensor *tensor) {
+  OM_DATA_TYPE dataType = omTensorGetDataType(tensor);
+  int64_t rank = omTensorGetRank(tensor);
+  const int64_t *shape = omTensorGetShape(tensor);
+  const int64_t *strides = omTensorGetStrides(tensor);
+  void *dataPtr = omTensorGetDataPtr(tensor);
+
+  printf("\trank = %lld\n", (long long)rank);
+  printf("\tdataType = %s\n", getDataTypeName(dataType));
+
+  if (rank == 1) {
```

Comment:
would it be possible to add at least a 2D tensor too? If the format is Numpy arrays, that makes for easy integration to Jupyter notebook testing.

Reply:
Fair request. I'll work on that.

Reply:
Done, we can now print `OMTensor` up to rank <= 3.